src/votekit/election_types.py

```diff
@@ -1,98 +1,125 @@
 from .profile import PreferenceProfile
 from .ballot import Ballot
-from .models import Outcome
+from .election_state import ElectionState
 from typing import Callable
 import random
 from fractions import Fraction
 from copy import deepcopy
 
-
+##We're not passing winner_votes to outcome right now
+# If we want to track which candidate each ballot is ultimately going to, we need to do more.
+# TODO:
+# 1. winner_votes to election_state is supposed to have all the ballots in the
+# initial profile that ended up going to any elected candidate.
+# Bit tricky since all of our methods work with current state of profile not the initial
+# 2. integrate Cincinatti transfer
 class STV:
     def __init__(self, profile: PreferenceProfile, transfer: Callable, seats: int):
-        self.profile = profile
-        self.transfer = transfer
-        self.elected: set = set()
-        self.eliminated: set = set()
-        self.seats = seats
-        self.threshold = self.get_threshold()
+        self.transfer: Callable = transfer
+        self.seats: int = seats
+
+        fp_votes = compute_votes(profile.get_candidates(), profile.get_ballots())
+        fp_order = [
+            y[0] for y in sorted(fp_votes.items(), key=lambda x: x[1], reverse=True)
+        ]
```

Comment:
these lines get repeated a few times, lets turn them into a order_cands() method that we can call when initializing election state and during the actual running of STV
`
ElectionState(
            curr_round=0, elected=[], eliminated=[], remaining=self.order_cands(), profile=profile
        )
`